src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionEntity.java

```diff
@@ -75,18 +79,6 @@ public class TranscriptionEntity extends CreatedModifiedBaseEntity {
     @Column(name = "end_ts")
     private OffsetDateTime end;
 
-    @Column(name = "requested_by")
```

Comment:
I'm not entirely sure why we are removing these columns - I remember we added them to the LLD a few months ago when it was being thought that we would need a way to do Transcription searches for a particular user view e.g. My Transcriptions (for a requestor, approver, transcriber) was the idea.
Although it looked like Jack thought the `approved_by` field could be changed to be `actioned_by` perhaps which could be used where the approver chose `Rejected` action instead.

@jackmaloney should @barry-jones50410 have removed these columns with the introduction of the new transcription_workflow table, to remove workflow related elements from transcription? How do we find this stuff now

Reply:
My understanding is that now the workflow table will be used to do these lookups. For example, to work out who is assigned to a transcription request, we would need to look at the workflow table and check which user performed the 'with transcriber' (or whatever we call it) task under the workflow_stage field. 

Let me know if any concerns with this approach.

Reply:
@jackmaloney My main concern is the fields are missing unless we are thinking we should be able to use the jpa auditing fields for a table e.g. created_by / last_modified_by which we're not able to populate currently in our schema. I think this table sounds like we'd only ever be inserting into the workflow table.

It's a bit different now than e.g. media_request which would use the `requestor` field and what is new recently `current_owner`. We also have a conflicting field of same name under transcription table - but that must be a legacy throw away.

Reply:
When I spoke to Barry yesterday about this, he advised the created_by field within the transcription_workflow table will be used for this purpose. 
The requestor column is used for legacy cases. Barry said "just for completeness, i ought to mention that the table currently also has a requestor column, which exists to support migration, if we subsequently determine that we cannot create an appropriate transcription_workflow record".